app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,17 @@
+class ReviewsController < ApplicationController
+
+  def create
+    @review = Review.new(review_params)
+    @review.book_id = params[:book_id]
+
+    @review.save
+
+    redirect_to book_path(@review.book)
+  end
+
+  private
+
+  def review_params
+    params.require(:review).permit(:body, :rating)
+  end
+end
```

Comment:
Pay careful attention to your user stories. None of them ask to be able to create a view from a book. It also says at the end of the spec to not build off a current_user to create a new form.